witchcraft-server/src/service/request_log.rs

```diff
@@ -119,25 +120,25 @@ where
         };
 
         let mdc = mdc::snapshot();
-        let uid = mdc
-            .safe()
-            .get(logging::mdc::UID_KEY)
-            .and_then(|v| UserId::deserialize(v.clone()).ok());
-        let sid = mdc
-            .safe()
-            .get(logging::mdc::SID_KEY)
-            .and_then(|v| SessionId::deserialize(v.clone()).ok());
-        let token_id = mdc
-            .safe()
-            .get(logging::mdc::TOKEN_ID_KEY)
-            .and_then(|v| TokenId::deserialize(v.clone()).ok());
         let trace_id = mdc
             .safe()
             .get(logging::mdc::TRACE_ID_KEY)
             .and_then(|v| TraceId::deserialize(v.clone()).ok());
 
         let mut params = vec![];
 
+        let unverified_jwt = req.extensions().get::<UnverifiedJwt>();
+        let uid = unverified_jwt.map(|v| UserId(v.unverified_user_id().to_string()));
+        let sid = unverified_jwt
+            .and_then(UnverifiedJwt::unverified_session_id)
+            .map(|v| SessionId(v.to_string()));
+        let token_id = unverified_jwt
+            .and_then(UnverifiedJwt::unverified_token_id)
+            .map(|v| TokenId(v.to_string()));
+        if let Some(org_id) = unverified_jwt.and_then(UnverifiedJwt::unverified_organization_id) {
+            params.push(("_org".to_string(), Any::new(org_id).unwrap()));
```

Comment:
Is the org safe here? What's the reason for excluding from the MDC?

Reply:
Yeah, it's just a UUID. Not sure why we want to keep it out of service logs - @carterkozak would know. The behavior here mirrors the change we made to WC-Java.

Reply:
service logs have a `_requestId` reference, so I opted to avoid data duplication. No harm in including this with service logs, just additional bytes to write/transfer/store/index.